css/style.css

```diff
@@ -1,3 +1,152 @@
+body{
+  font-family: 'Open Sans', sans-serif;
+}
+
+header{
+  background: url(../images/header_bg.jpg);
+  background-repeat: no-repeat;
+  background-size: cover;
+  height: 850px;
```

Comment:
@ValentinaChung you should change the height smaller, so that it matches the png. :hammer: